### ISSUE TITLE
Add tests in Django

### ISSUE THREAD

**Issue by cfahlgren1** [cfahlgren1/Bounce#4]
Add tests for CI/CD Pipeline for API, Model,End to End testing with cypress.io
Use django-nose or coverage to track testing coverage

**Comment by johnckealy** [comment]
Hi, I'd be down to take this one if that's okay? I'll write as many tests as I can; though there'll always be more add.

Also getting some errors setting up the dev environment. +1 for writing that setup guide :)

**Comment by cfahlgren1** [comment]
Yes, for sure. One of the main things before the majority of development can be done is adding tests. What errors are you getting? The hard part is setting up the Geodjango libraries. The project setup follows https://realpython.com/location-based-app-with-geodjango-tutorial/ almost step for step.
